### PR TITLE
LPS-189219 Popover body (small modal) from Beta Feature doesn't is minimize

### PR DESCRIPTION
**Pull request by kildarep**
Hello Reviewer!

Related Issue: https://liferay.atlassian.net/browse/LPS-189219

### CODE REVIEW

File: modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BetaButton.tsx [hunk 1]
```diff
@@ -30,11 +32,12 @@ export function BetaButton() {
 					alignPosition="top"
 					disableScroll
 					header={Liferay.Language.get('beta-feature')}
-					show={true}
+					show={showPopover}
 					trigger={
 						<ClayButton
 							className="rounded-circle"
 							displayType="beta"
+							onClick={() => setShowPopover(!showPopover)}
```
Comment (danvb112) [inline modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BetaButton.tsx:40]: @kildarep, as we are going to remove the Tooltip we can show the Popover on the hover of the button, for that we can remove the `onClick` and add two attributes called `onMouseOver` and `onMouseOut`:

```tsx
onMouseOver={() => setShowPopover(true)}
onMouseOut={() => setShowPopover(false)}
```

**Comment by kildarep** [comment]
Hi @danvb112, Modified for suggested changes, thanks for the suggestions :wink:

**Comment by kildarep** [comment]
ci:test:sf

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward:force

**Pull request closed without merging**